docs/architecture.svg

```diff
@@ -1,282 +1,3 @@
000000" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 937.19 322.04 L 944.18 318.54 L 942.44 322.04 L 944.19 325.54 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><path d="M 1011.52 115.79 L 1015.02 122.79 L 1011.52 121.04 L 1008.02 122.79 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><path d="M 653 189.18 Q 1011.58 189.17 1011.57 313.19" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 647.75 189.18 L 654.75 185.68 L 653 189.18 L 654.75 192.68 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><path d="M 1011.57 318.44 L 1008.07 311.44 L 1011.57 313.19 L 1015.07 311.44 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><rect x="495.62" y="170.55" width="151.01" height="37.25" fill="#a9c4eb" stroke="#000000" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 149px; height: 1px; padding-top: 189px; margin-left: 497px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 14px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">spyglass</div></div></div></foreignObject><text x="571" y="193" fill="#000000" font-family="Helvetica" font-size="14px" text-anchor="middle">spyglass</text></switch></g><path d="M 464.23 214.01 Q 477.92 214 477.92 196.96 Q 477.92 179.92 489.25 179.88" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 458.98 214.01 L 465.98 210.51 L 464.23 214.01 L 465.99 217.51 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><path d="M 494.5 179.87 L 487.51 183.39 L 489.25 179.88 L 487.49 176.39 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><path d="M 856.97 263.52 Q 857 201.58 833.58 201.58 Q 810.17 201.58 810.22 139.66" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 856.97 268.77 L 853.47 261.77 L 856.97 263.52 L 860.47 261.77 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><path d="M 810.22 134.41 L 813.72 141.42 L 810.22 139.66 L 806.72 141.41 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
Can you make this have line breaks for ease of diffing or review?

Reply:
Updated in 9bffcca89d.
This appears to be something with draw.io's exporting